src/keydra/clients/splunk.py

```diff
@@ -66,45 +66,21 @@ def __init__(self, username, password, host, verify, port=8089):
 
         self._instance = host.split('.')[0]
 
-    def _get(self, url, params={}):
+    def _get(self, url: str, params: dict) -> dict:
```

Comment:
The only concern is that we're changing from an optional argument to a required argument so there might be assumptions somewhere else where it assumes an optional argument.

Reply:
Good point, these are private helper methods of the Splunk client class, so there are no external invocations.